src/message.rs

```diff
@@ -20,7 +20,7 @@ impl Ord for Message {
         (&self.filename, self.location.row(), self.location.column()).cmp(&(
             &other.filename,
             other.location.row(),
-            self.location.column(),
+            other.location.column(),
```

Comment:
Oof, thanks!

Reply:
@charliermarsh I opened https://github.com/charliermarsh/ruff/pull/152 just to fix this line.